.travis.yml

```diff
@@ -22,10 +26,11 @@ before_install:
   - conda update -q conda
 
 install:
-  - conda create -n testenv --yes -c quantopian pip python=$TRAVIS_PYTHON_VERSION numpy=$NUMPY_VERSION pandas=$PANDAS_VERSION scipy=$SCIPY_VERSION libgfortran=$LIBGFORTRAN_VERSION
+  - conda create -n testenv --yes -c quantopian pip python=$TRAVIS_PYTHON_VERSION pandas=$PANDAS_VERSION
   - source activate testenv
-  - CACHE_DIR="$HOME/.cache/.pip/pip_np""$CONDA_NPY"
+  - CACHE_DIR="$HOME/.cache/.pip/pip_np"
```

Comment:
Any reason to specify the cache dir? Also, the `np` suffix for numpy is inaccurate now.

Reply:
fixed